src/lib/gw2/itemstats.spec.js

```diff
@@ -105,6 +181,636 @@ const SELECTABLE_RING = {
   },
 };
 
+const SELECTABLE_LEGEND_ARMOR_CHEST = {
```

Comment:
can we move the test data into a separate file? this test is getting really long because of them :)